bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/LedgerUnderreplicationManager.java

```diff
@@ -148,6 +148,18 @@ void enableLedgerReplication()
     boolean isLedgerReplicationEnabled()
             throws ReplicationException.UnavailableException;
 
+    /**
+     * Receive notification asynchronously when the ledger replication process
+     * is changed.
+     *
+     * @param cb
+     *            - callback implementation to receive the notification
+     */
+    default void notifyLedgerReplicationStatusChanged(GenericCallback<Void> cb)
+            throws ReplicationException.UnavailableException {
+        throw new RuntimeException("not implemented");
```

Comment:
This would be breaking the (runtime) compatibility for an existing `LedgerUnderReplicationManager` that does not implement the method, since it will throw exception.

Reply:
Remove default  method as current now.